### ISSUE TITLE
Pi video out quits mid song

### ISSUE THREAD

**Issue by erixsparhawk** [SpottyMatt/raspian-3b-stepmania-arcade#18]
Just got this setup on a pi 4 as per all your instructions. I've just been playing the songs that came with the build and sometimes mid song the video output of the pi stops but the program and song keep going. I can't get the video back on unless I restart the Pi. Any thoughts what might be going on? I'm thinking some sort of key binding issue. I've got u,d,l,r, start and back bound to my matt. Video issues seem to happen in the middle of lots of two footed jumping.

**Comment by misholson** [comment]
I think I'm having a similar problem. I lose video after a jump occasionally. My Linux skills are real rusty so I'm not even sure where to look to start troubleshooting.

**Comment by erixsparhawk** [comment]
I solved the problem by manually disabling the screen saver function. I think I used this, but I already can't remember: https://www.raspberrypi.org/documentation/configuration/screensaver.md

**Comment by misholson** [comment]
Yes this was the issue, thank you! I didn't even realize that it was just the video because my TV just started showing a "No input" error like what it would say if it was turned all the way off.

**Comment by SpottyMatt** [comment]
The arcade setup should be updated to include disabling screensaver, if enabled.
